azurestack/resource_arm_virtual_machine_scale_set.go

```diff
@@ -442,10 +442,10 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 						},
 
 						"managed_disk_type": {
-							Type:          schema.TypeString,
-							Optional:      true,
-							Computed:      true,
-							ConflictsWith: []string{"storage_profile_os_disk.vhd_containers"},
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							// ConflictsWith: []string{"storage_profile_os_disk.vhd_containers"},
```

Comment:
Would you mind updating this so that it is deleted rather than commented out? 

Also could you update the commit message? Just something like 
"Remove invalid ConflictsWith attribute from Managed Disk resource."